### ISSUE TITLE
Error -4058 can't find idris executable

### ISSUE THREAD

**Issue by GSmithApps** [idris-hackers/atom-language-idris#220]
I'm getting error message -4058 that says it can't find the idris executable. I have the right file path in my package settings, so I don't know what the error is. Here's the message:

Thanks!

![image](https://user-images.githubusercontent.com/44339825/47995169-740bae80-e0c2-11e8-8d89-d32855c64806.png)

**Comment by justjoheinz** [comment]
I have never worked on Windows - could you try to specify the path including the `.exe` ending?

**Comment by GSmithApps** [comment]
That's a good idea. I'm pretty sure I tried it, and it didn't work.  But I'll try again and let you know.  Thank you!

**Comment by justjoheinz** [comment]
Or play around with the slashes, like `C:\...` or `C:\\...` ?

**Comment by GSmithApps** [comment]
I originally just went to the file's properties and copied the whole file path. I assumed that was kind of a foolproof method. I'll try messing with the slashes

**Comment by GSmithApps** [comment]
yeah I just tried and it didn't work:(

**Comment by justjoheinz** [comment]
Can you invoke idris from the terminal?

**Comment by justjoheinz** [comment]
Or a Anti Virus software denies to spawn the idris process?

**Comment by GSmithApps** [comment]
Let me check. Those are good ideas! Thank you very much for helping

**Comment by GSmithApps** [comment]
yeah i can invoke it from the prompt.  I'll look at the antivirus thing

**Comment by GSmithApps** [comment]
still not working

**Comment by justjoheinz** [comment]
@melted Could you post the exact entry of your idris path setting for reference please?

**Comment by joelberkeley** [comment]
I'm seeing the same on ubuntu 18.04 but with error code -2. idris is installed at ~/.cabal/bin/idris and runs fine in the terminal
